src/styles/blocks/catalog.scss

```diff
@@ -18,8 +18,15 @@
     border-radius: 5px;
     margin-bottom: 48px;
 
+    transition: 0.3s linear;
```

Comment:
Add transition props, don't rely on the default `all` value.
Add duration to variables.